shoes-core/spec/shoes/animation_spec.rb

```diff
@@ -60,7 +60,7 @@
   it { is_expected.not_to be_stopped }
 
   describe "with framerate" do
-    let(:opts) { {:framerate => 36, :app => app} }
+    let(:opts) { { :framerate => 36, :app => app } }
```

Comment:
not sure about the spacious usage of whitespace in hash brackets either - I remember we talked about this, does someone know what we decided? @jasonrclark